source/TextBlade.Plateform.Windows/TextBlade.Plateform.Windows.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
I think you spelled `Platform` with an extra `e`

Reply:
Ah yes... One of my many failings is my inability to spell in any language :P